From 5dcbe9ff67052c5309016b39593b26755d9168bc Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:23 -0400
Subject: [redhat] kabi: Adding symbol param_get_charp (kernel/params.c)

Message-id: <70b8469050cfde027f9264f83dede078f7c16a1c.1410510773.git.poros@redhat.com>
Patchwork-id: 95941
O-Subject: [PATCH RHEL7.1 143/233] kabi: Adding symbol param_get_charp (kernel/params.c)
Bugzilla: 1071098
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Not used directly, used by module_param internally for type char.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_get_charp
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_get_charp
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_get_charp

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_get_charp b/redhat/kabi/kabi-module/kabi_ppc64/param_get_charp
new file mode 100644
index 0000000..a2032ef
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_get_charp
@@ -0,0 +1,2 @@
+#1-
+0xa250c838 param_get_charp vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_get_charp b/redhat/kabi/kabi-module/kabi_s390x/param_get_charp
new file mode 100644
index 0000000..a2032ef
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_get_charp
@@ -0,0 +1,2 @@
+#1-
+0xa250c838 param_get_charp vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_get_charp b/redhat/kabi/kabi-module/kabi_x86_64/param_get_charp
new file mode 100644
index 0000000..a2032ef
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_get_charp
@@ -0,0 +1,2 @@
+#1-
+0xa250c838 param_get_charp vmlinux EXPORT_SYMBOL
-- 
1.7.1